packages/components/src/__actions__/LinkButton/v3/_docs/LinkButton.doc.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react'
+import { Meta, StoryObj } from '@storybook/react'
+import { VisuallyHidden } from '~components/VisuallyHidden'
+import { Icon } from '~components/__future__'
+import { ReversedColors } from '~components/__utilities__/v3'
+import { LinkButton } from '../index'
+
+const meta = {
+  title: 'Actions/LinkButton/LinkButton (v3)',
+  component: LinkButton,
+  args: {
+    children: 'Label',
+    href: '#link-button-clicked',
+  },
+  argTypes: {
+    icon: {
+      options: ['delete', 'arrow', 'plus'],
+      mapping: {
+        delete: <Icon isPresentational name="delete" />,
+        arrow: <Icon isPresentational name="arrow_forward" />,
+        add: <Icon isPresentational name="add" />,
+      },
+    },
+  },
+} satisfies Meta<typeof LinkButton>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {}
+
+export const LinkButtonOpensInNewTab: Story = {
+  args: {
+    children: (
+      <>
+        Label
+        <VisuallyHidden> opens a new tab</VisuallyHidden>
+      </>
+    ),
+    href: 'https://www.google.com',
+    target: '_blank',
+    rel: 'noopener noreferrer',
+    icon: <Icon isPresentational name="open_in_new" shouldMirrorInRTL />,
+    iconPosition: 'end',
+  },
+}
+
+export const LinkButtonVariants: Story = {
+  render: ({ children: _, ...otherArgs }) => (
+    <>
+      <LinkButton {...otherArgs} variant="primary">
+        Primary
+      </LinkButton>
+      <LinkButton {...otherArgs} variant="secondary">
+        Secondary
+      </LinkButton>
+      <LinkButton {...otherArgs} variant="tertiary">
+        Tertiary
+      </LinkButton>
+    </>
+  ),
+  decorators: [
+    (Story) => (
+      <div className="flex gap-8">
+        <Story />
+      </div>
+    ),
+  ],
+}
+
+export const LinkButtonVariantsReversed: Story = {
+  render: (args) => (
+    <ReversedColors isReversed={true}>
+      <LinkButton {...args} variant="primary" />
+      <LinkButton {...args} variant="secondary" />
+      <LinkButton {...args} variant="tertiary" />
+    </ReversedColors>
+  ),
+  parameters: {
+    reverseColors: true,
+  },
```

Comment:
This should work right? I think the global decorator using the `reverseColors` param should inject the context. Maybe even would mean you could remove your custom decorator.

```suggestion
export const LinkButtonVariantsReversed: Story = {
  ...LinkButtonVariants,
  parameters: {
    reverseColors: true,
  },
```

Reply:
Noice, yeah don't know why I originally did it that way. This is much nicer :)